includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -578,55 +443,257 @@ protected function wc_setup_wcs_tout() {
 	}
 
 	/**
+	 * Helper method to queue the background install of a plugin.
+	 *
+	 * @param string $plugin_id  Plugin id used for background install.
+	 * @param array  $plugin_info Plugin info array containing at least main file and repo slug.
+	 * @param bool   $background Optional. Whether or not to queue the install in the backgroud.
+	 */
+	protected function install_plugin( $plugin_id, $plugin_info, $background = true ) {
+		if ( ! empty( $plugin_info['file'] ) && is_plugin_active( $plugin_info['file'] ) ) {
+			return;
+		}
+
+		if ( $background ) {
+			wp_schedule_single_event( time() + 10, 'woocommerce_plugin_background_installer', array( $plugin_id, $plugin_info ) );
+		} else {
+			WC_Install::background_installer( $plugin_id, $plugin_info );
+		}
+	}
+
+	/**
+	 * Helper method to install Jetpack.
+	 *
+	 * @param bool $now Optional. Whether or not to queue the install.
+	 */
+	protected function install_jetpack( $now = false ) {
+		$this->install_plugin( 'jetpack', array(
+			'file'      => 'jetpack/jetpack.php',
+			'name'      => __( 'Jetpack', 'woocommerce' ),
+			'repo-slug' => 'jetpack',
+		), ! $now );
+
+		if ( ! $now ) {
+			update_option( 'woocommerce_setup_queued_jetpack_install', true );
+		}
+	}
+
+	/**
+	 * Helper method to install WooCommerce Services and its Jetpack dependency.
+	 */
+	protected function install_woocommerce_services() {
+		$this->install_jetpack();
+		$this->install_plugin( 'woocommerce-services', array(
+			'file'      => 'woocommerce-services/woocommerce-services.php',
+			'name'      => __( 'WooCommerce Services', 'woocommerce' ),
+			'repo-slug' => 'woocommerce-services',
+		) );
+	}
+
+	/**
+	 * Get the WCS shipping carrier for a given country code.
+	 *
+	 * Can also be used to determine if WCS supports a given country.
+	 *
+	 * @param $country_code
+	 * @return bool|string Carrier name if supported, boolean False otherwise.
+	 */
+	protected function get_wcs_shipping_carrier( $country_code ) {
+		switch ( $country_code ) {
+			case 'US':
+				return 'USPS';
+			case 'CA':
+				return 'Canada Post';
+			default:
+				return false;
+		}
+	}
+
+	/**
+	 * Get shipping methods based on country code.
+	 *
+	 * @param $country_code
+	 * @return array
+	 */
+	protected function get_wizard_shipping_methods( $country_code ) {
+		$shipping_methods = array(
```

Comment:
`Local pickup` is missing, I don't know if it's intentional.

Reply:
cc: @olaolusoga

Reply:
It wasn't. thanks for the catch. We should include.

Reply:
On second thought - we shouldn't include this. We're setting up either a zone that includes the entire country the store is located in, or all others. I don't think local pickup makes sense.